### PR TITLE
[FlexiDAG] fix force upgrade test

### PR DESCRIPTION
**Pull request by welbon**
## Pull request type

Please check the type of change your PR introduces:

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] Documentation content changes
- [ ] Other (please describe):

## What is the current behavior?

Issue Number: N/A

## What is the new behavior?

-
-
-

### CODE REVIEW

File: scripts/build_stdlib.sh
Comment (simonjiao) [inline scripts/build_stdlib.sh:1]: where the `generated` genesis will be placed? The proper directory should be genesis\generated.

File: vm/vm-runtime/src/force_upgrade_management.rs [hunk 1]
```diff
@@ -9,7 +9,7 @@ pub const FORCE_UPGRADE_BLOCK_NUMBER: u64 = 17500000;
 
 pub fn get_force_upgrade_block_number(chain_id: &ChainId) -> u64 {
     if chain_id.is_test() {
-        50
+        10
```
Comment (simonjiao) [inline vm/vm-runtime/src/force_upgrade_management.rs:12]: Please dont change this height smaller. It could cause some test cases failed.

**Review by simonjiao — CHANGES_REQUESTED** [review]

File: vm/vm-runtime/src/force_upgrade_management.rs (hunk 1 above)
Comment (welbon) [reply vm/vm-runtime/src/force_upgrade_management.rs:12]: ok

**Review by simonjiao — APPROVED** [review]
LGTM

**Pull request merged**